apis/apps/v1alpha1/servicedescriptor_types.go

```diff
@@ -47,20 +47,27 @@ type ServiceDescriptorSpec struct {
 	// +kubebuilder:validation:Required
 	ServiceVersion string `json:"serviceVersion"`
 
-	// Specifies the URL or IP address of the external service.
+	// Specifies the endpoint of the external service.
+	//
+	// If the service is exposed via a cluster, the endpoint will be provided in the format of <host>:<port>.
 	//
 	// +optional
 	Endpoint *CredentialVar `json:"endpoint,omitempty"`
```

Comment:
I guess that the variables of service-ref are all from the connection credential originally.

Reply:
> I guess that the variables of service-ref are all from the connection credential originally.

Yes, the initial version of serviceReference obtained all its information from the Connection Credential, thus the definition of CredentialVar was reused. In subsequent versions, the name of CredentialVar can be changed since it currently includes the generic definitions of Value and ValueFrom.